site/ja/tutorials/load_data/tfrecord.ipynb

```diff
@@ -352,7 +352,7 @@
         "このデータセットには4つの特徴量があります。\n",
         "- `False` または `True` を表す論理値。出現確率は等しいものとします。\n",
         "- ランダムなバイト値。全体において一様であるとします。\n",
-        "- `[-10000, 10000)` の範囲から一様にサンプリングした整数値。\n",
+        "- `[-0, 5)` の範囲から一様にサンプリングした整数値。\n",
```

Comment:
[Strong suggestion]
Based on https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb, `[-0, 5)` -> `[0, 5]`

Reply:
Could you please add a translation of this sentence `a string feature generated from a string table by using the integer feature as an index`?

Reply:
Thank you for your comments.
I think `[0, 5]` should be `[0, 5)`  since original one is wrong.